day03/deploy_django_app.sh

```
@@ -1,74 +1,38 @@
-    # sudo systemctl enable docker
-    # sudo systemctl enable nginx
-    # sudo systemctl restart docker
+restarts() {
+    sudo chown "$USER" /var/run/docker.sock 
+    sudo systemctl enable docker
+    sudo systemctl enable nginx
+    sudo systemctl restart docker
 }
 
-# Function to deploy the Django app
 deploy() {
```

Shellcheck (0.11.0)

[error] Couldn't parse this brace group. Fix to allow more checks.

(SC1073)